index.html

```diff
@@ -0,0 +1,159 @@
+            <p>
+              The three days of the conference will contribute to build
+              long-term profitable and sustainable communication among the
+              participants in a wide variety of scientific areas with the common
+              interest in Applied Science, Engineering and Technology.
+              </p>
+            </div>
+            <div class="under-content">
+            </p>
+            <h2>2023.03.23(TUE) ~ 25(WED)</h2>
+            <p>&copy SHAGRI-LA HOTEL, BANGKOK, THAILAND</p>
+          </div>
+          
+        </div>
+      </section>
+    </section>
+
+    
+    <section class="Section-4">
+
+      <h3 class="centering" id="underline">Participation Opportunities
+       
+      </h3>
+
+      <div class="col">
+        <div class="row">
+          <img src="images/listener.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
+        <div class="row">
+          <img src="images/poster-presentation.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
+        <div class="row">
+          <img src="images/workshop-presentation.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
+        <div class="row">
+          <img src="images/poster-presentation.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
+        <div class="row">
+          <img src="images/workshop-presentation.png" alt="">
+          <h4>Oral Presentaion</h4>
+          <p>Oral Presentation is an effective way to present your valuable research work to get recognized.</p>
+        </div>
+      </div>
+      <a href="HomePage.html" id="centering" class="link">SEE THE WHOLE PROGRAM</a>
+
+      
+
+     
+    </section>
+     <!-- section 5 -->
+     <section class="Professors">
+      <div class="container speakers-section-info">
+        <h2>Keynote Speakers</h2>
+        <div class="cardContainer"></div>
+      </div>
+    </section>
+    
+    <section class="Section-6">
+      <h3 class="centering">Partners</h3>
+      <div class="image-container">
+        <img src="images/airbnb-logo.png" alt="">
+        <img src="images/mozila.png" alt="">
+        <img src="images/google.png" alt="">
+        <img src="images/naver.png" alt="">
+      </div>
+    </section>
+
+    <section class="Section-7">
+      <div>
+      <h3>Organized by</h3>
+      <img src="images/logo.png" alt="" class="logo" />
+    </div>
+      <div>
+      <p>	&copy GSASET2023 All rights reserved.</p>
+    </div>
+    </section>
```

Comment:
Kindly use the html footer tag for this section. Also make sure that the design for both desktop and mobile versions of the home and about pages matches the actual design. Kindly check and fix some of the disparities below, this include the colors, margins, paddings etc. 

## Desktop

### _Home page_

Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-27 13-25-03](https://user-images.githubusercontent.com/65233860/198298353-4c5b0b47-a130-4a26-ae48-bc7c394206cb.png)    |  ![Screenshot from 2022-10-27 13-27-10](https://user-images.githubusercontent.com/65233860/198298402-8109b078-90b3-4d5c-9b65-945e3a55366c.png)

### _About page_

Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-27 13-25-37](https://user-images.githubusercontent.com/65233860/198298782-4352f9ca-9048-47c1-9524-571af9fe67c7.png)  |  ![Screenshot from 2022-10-27 13-27-21](https://user-images.githubusercontent.com/65233860/198298833-c7d97d99-53b4-4991-8945-0c6e300bc8ed.png)

## Mobile

### _Home page_

Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-27 13-25-03](https://user-images.githubusercontent.com/65233860/198299154-28625ae6-22f4-4b24-b90b-7c56a4111e08.png)  |  ![Screenshot from 2022-10-27 13-27-32](https://user-images.githubusercontent.com/65233860/198299193-59c029bf-37ef-47b4-95a7-821b0e5fd1b9.png)


### _About page_

Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-27 13-26-12](https://user-images.githubusercontent.com/65233860/198299337-98094459-1bfe-420f-bd08-35dd36c7ce14.png)  |  ![Screenshot from 2022-10-27 13-27-38](https://user-images.githubusercontent.com/65233860/198299395-6169f345-cfe0-4074-bb80-2860a9eab316.png)